markdowns/02_Finetune_a_model_on_your_data.md

```diff
@@ -63,102 +59,73 @@ logging.getLogger("haystack").setLevel(logging.INFO)
+data_dir = "data/fine-tuning"
 
-Sometimes, however, you can't use an already distilled model and have to distil it yourself. For this case, haystack has implemented [distillation features](https://haystack.deepset.ai/guides/model-distillation).
 
-### Augmenting your training data
-To get the most out of model distillation, we recommend increasing the size of your training data by using data augmentation. You can do this by running the [`augment_squad.py` script](https://github.com/deepset-ai/haystack/blob/main/haystack/utils/augment_squad.py):
+fetch_archive_from_http(
+    url="https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-downstream/squad20.tar.gz",
+    output_dir=data_dir
+)
+```
+
+3. Train the model with yor won data and save it to "my_model"
 
 
 ```python
-# Downloading script
-!wget https://raw.githubusercontent.com/deepset-ai/haystack/main/haystack/utils/augment_squad.py
 
-glove_dir = "data/tutorial2/gloves"
-squad_dir = "data/tutorial2/squad_small"
+reader.train(data_dir=data_dir, train_filename="squad20/dev-v2.0.json", use_gpu=True, n_epochs=1, save_dir="my_model")
+```
 
-# Downloading smaller glove vector file (only for demonstration purposes)
-glove_url = "https://nlp.stanford.edu/data/glove.6B.zip"
-fetch_archive_from_http(url=glove_url, output_dir=glove_dir)
+4. Now initialize a new reader with your fine-tuned model
 
-# Downloading very small dataset to make tutorial faster (please use a bigger dataset for real use cases)
-s3_url = "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-qa/datasets/documents/squad_small.json.zip"
-fetch_archive_from_http(url=s3_url, output_dir=squad_dir)
 
-# Just replace the path with your dataset and adjust the output (also please remove glove path to use bigger glove vector file)
-!python augment_squad.py --squad_path data/tutorial2/squad_small/squad_small.json --output_path augmented_dataset.json --multiplication_factor 2 --glove_path data/tutorial2/gloves/glove.6B.300d.txt
+```python
+new_reader = FARMReader(model_name_or_path="my_model")
 ```
 
-In this case, we use a multiplication factor of 2 to keep this example lightweight. Usually you would use a factor like 20 depending on the size of your training data. Augmenting this small dataset with a multiplication factor of 2, should take about 5 to 10 minutes to run on one V100 GPU.
-
-### Running distillation
-Distillation in haystack is done in two steps: First, you run intermediate layer distillation on the augmented dataset to ensure the two models behave similarly. After that, you run the prediction layer distillation on the non-augmented dataset to optimize the model for your specific task.
-
-If you want, you can leave out the intermediate layer distillation step and only run the prediction layer distillation. This way you also do not need to perform data augmentation. However, this will make the model significantly less accurate.
+5. Finaly, try using the `new_reader` that was initialized with your fine-tuned model.
```

Comment:
This sounds like we're unsure it's going to work. I'd rephrase: Finally, use the `new_reader` that was initialized with...